executor/distsql.go

```diff
@@ -139,9 +141,15 @@ func zone(sctx sessionctx.Context) (string, int64) {
 	if name == "Local" {
 		path, err := filepath.EvalSymlinks("/etc/localtime")
 		if err != nil {
+			log.Errorln(err)
 			return "Sysytem", int64(offset)
```

Comment:
`System`?

Reply:
Yes. I have no idea why this happens.  - - Already fixed.